pkgs/development/go-modules/generic/default.nix

```diff
@@ -129,6 +131,9 @@ let
 
     dontFixup = true;
   }; in modArgs // (
+    if vendorSha256 == lib.symbols.NOCHROOT then
+      { __noChroot = true; }
```

Comment:
Is `__noChroot` permitted on ofborg and hydra? If so, you may want to add a test for this.

It might be ok to leave this in "unsupported" territory.

Reply:
It depends on the hydra instance configuration. Is there a way to test this at eval time?